src/v0/destinations/facebook_conversions/data/FBCUserDataConfig.json

```diff
@@ -86,15 +86,15 @@
   },
   {
     "destKey": "ct",
-    "sourceKeys": ["traits.address.city", "context.traits.address.city"],
+    "sourceKeys": ["traits.address.city", "context.traits.address.city", "context.geo.city"],
     "required": false,
     "metadata": {
       "type": ["toLower", "hashToSha256"]
     }
   },
   {
     "destKey": "st",
-    "sourceKeys": ["traits.address.state", "context.traits.address.state"],
+    "sourceKeys": ["traits.address.state", "context.traits.address.state", "context.geo.region"],
```

Comment:
@krishna2020 @sandeepdsvs @utsabc 

Should we do this mapping ?

Example payload in context
```json
"geo": {
      "city": "Gurugram",
      "country": "IN",
      "ip": "223.190.82.63",
      "location": "28.459700,77.028200",
      "postal": "122001",
      "region": "Haryana",
      "timezone": "Asia/Kolkata"
    },
    "ip": "223.190.82.63"
```